### ISSUE TITLE
sematic tags and  iframe/body tag on CSS

### ISSUE THREAD

**Issue by dasileker** [oracleot/embedding-images-and-videos#19]
@Diego lira

_**Comment:**_ 

> Try to mix the use of semantic and no semantic tags, as a suggestion use it to structure your layout highlighting the most important parts especially for a screen readers to know what they should be looking for.

@Daniel Ronan

> My comment was for the iframe/body tag on CSS, you can use the ‘*’ instead to apply for all of your HTML and I tend to use, box-sizing: border-box, padding: 0 and margin:0 so that it takes off the automatic sizing and then you will be able to size and position it yourself.
